wgpu-core/src/device/queue.rs

```diff
@@ -143,12 +143,56 @@ crate::impl_storage_item!(Queue);
 impl Drop for Queue {
     fn drop(&mut self) {
         resource_log!("Drop {}", self.error_ident());
+
+        let last_successful_submission_index = self
+            .device
+            .last_successful_submission_index
+            .load(Ordering::Acquire);
```

Comment:
todo: I want to double-check atomic orderings again, just to make sure I feel confident with this. I don't trust myself to eyeball this. 😅

Reply:
Seems right to me, resolving!